src/app/myworkflows/sidebar-accordion/github-apps-logs/github-apps-logs.component.html

```diff
@@ -2,7 +2,13 @@ <h1 mat-dialog-title>GitHub App Logs</h1>
 <div mat-dialog-content>
   <app-loading [loading]="loading">
     <mat-card *ngIf="showContent === 'error'" class="alert alert-warning mat-elevation-z" role="alert">
-      <mat-icon class="alert-warning-icon">warning</mat-icon> There were problems retrieving GitHub App logs for this organization.
+      <mat-icon class="alert-warning-icon">warning</mat-icon> There were problems retrieving the GitHub App logs for this organization.<br />
+      Typically, if you cannot view the logs for your GitHub user account, you must unlink and relink it in<br />
```

Comment:
This obviously happened, but is it "typical" as opposed to, say, "often"?
Open question, I don't have a good sense of what other failures occur aside from "no logs yet" (presumably wouldn't result in an error, just empty) or incorrect permissions ( https://docs.dockstore.org/en/stable/getting-started/github-apps/github-apps-troubleshooting-tips.html#github-app-permissions-faq occurred in the WDL workshop for example)